loops-and-arrays.js

```diff
@@ -53,3 +76,10 @@ const squares = [30, 2, 8, 24, 11];
 //   'Het volume van 24 is 13824',
 //   'Het volume van 11 is 1331'
 // ]
+
+
+const squares = [30, 2, 8, 24, 11];
+console.log(squares)
+for (let i = 0; i < squares.length; i++) {
+    console.log("Het volume van " + squares[i] + " is " + (Math.pow(squares[i],3)));
```

Comment:
Slim! Deze functie kende ik niet maar verkort het aanzienlijk. Daar is over nagedacht. De uitwerking zoals te zien is bij de uitwerkingen heb je ongetwijfeld gezien.